addons/im_livechat/static/src/components/composer/composer_tests.js

```diff
@@ -61,6 +61,24 @@ QUnit.test('livechat: no add attachment button', async function (assert) {
     );
 });
 
+QUnit.test('livechat: disable attachment upload via drag and drop', async function (assert) {
+    assert.expect(2);
+
+    await this.start();
+    const thread = this.env.models['mail.thread'].create({
+        channel_type: 'livechat',
+        id: 10,
+        model: 'mail.channel',
+    });
+    await this.createComposerComponent(thread.composer);
+    assert.containsOnce(document.body, '.o_Composer', "should have a composer");
+    assert.containsNone(
+        document.body,
+        '.o_Composer_dropZone',
+        "composer linked to livechat should not have a dropzone"
+    );
+});
+
```

Comment:
This test is ok for 14.0, but you are going to need to adapt this test to the recent change (pyEnv usage) between 15.0 and master.